README.md

```diff
@@ -19,7 +19,7 @@ To automate the following steps:
 ## Prerequisites
 
 - eXist 5.x
-- Current versions of ant, git, and bower executables
+- Current versions of ant, ant-contrib, git, and bower executables
```

Comment:
The ant-contrib dependency is already handled by the pom.xml. It shouldn't be listed here as a prerequisite.

Reply:
Fixed with https://github.com/evolvedbinary/hsg-project/pull/3/commits/6037e2b3dc231229baf50d6e691128e1fa6d1a60